features/support/journey_helpers.rb

```diff
@@ -41,13 +41,26 @@ def old_select_lower_tier_options
 end
 
 def select_random_upper_tier_options(carrier)
+  return answer_random_upper_tier_questions(carrier) unless @resource_object == :new_registration
+
+  i = rand(2)
+  if i == 0
+    @journey.check_your_tier_page.submit(option: :unknown)
+
+    answer_random_upper_tier_questions(carrier)
```

Comment:
Do we need another couple of lines here?
```suggestion
    answer_random_upper_tier_questions(carrier)
    # Proceed from "You need to register as an upper tier waste carrier":
    @journey.standard_page.submit
```